src/collective/cover/tests/test_banner_tile.robot

```diff
@@ -21,8 +24,8 @@ ${edit_link_selector}  a.edit-tile-link
 *** Test cases ***
 
 Test Banner Tile
-    # XXX: test is randomly failing under Plone 4.2
-    [Tags]  Mandelbug
+    # XXX: test is randomly failing under Plone 4.2 only
+    Run keyword if  '${CMFPLONE_VERSION}' >= '4.3'  Remove Tags  Mandelbug
```

Comment:
@datakurre this seems not to be working; what's the value of `${CMFPLONE_VERSION}` under Plone 4.2?